docker/services/docker-compose-l2geth.yml

```diff
@@ -1,13 +0,0 @@
-version: '3.8'
-
-services:
-  l2geth:
-    image: ${L2_DOCKER_TAG:-specularl2/l2geth:latest}
-    build:
-      context: ../..
-      dockerfile: ./clients/geth/specular/Dockerfile
-    volumes:
-      - ${SPECULAR_DATADIR:-$PWD/specular-datadir}:/specular
-    env_file:
-      - ../envs/common.env
-    entrypoint: geth.sh
```

Comment:
can we delete the script too? (if we delete both scripts can delete the whole dir)

Reply:
deleting this for now since it's not being used